src/lib/esnext.arraybuffer.d.ts

```diff
@@ -0,0 +1,58 @@
+interface ArrayBuffer {
+	/**
+	 * Read-only. The maximum length that this ArrayBuffer can be resized to (in bytes).
+	 */
+	readonly maxByteLength: number;
```

Comment:
I suggest using `get maxByteLength(): number` unless the team thinks otherwise; ditto for the other occurences.

Reply:
I've seen the `readonly` as well in multiple places. But yeah, let's wait for the team to comment on it.

Reply:
I agree with @Josh-Cena, the spec example uses accessors, so it makes sense to do so here.

(It's quite likely that old `readonly`s should be changed to accessors, but that might break backward compatibility, eg classes extend lib interfaces and override properties.)

Reply:
> I agree with @Josh-Cena, the spec example uses accessors, so it makes sense to do so here.
> 
> (It's quite likely that old `readonly`s should be changed to accessors, but that might break backward compatibility, eg classes extend lib interfaces and override properties.)

That seems like a breaking change we might want to take, though perhaps only if we can find a way to only enable it when `--useDefineForClassProperties` is `true` or the target it sufficiently high enough to warrant it.